Library/Formula/freeimage.rb

```diff
@@ -26,6 +26,13 @@ class Freeimage < Formula
 
   patch :DATA
 
+  # fix GCC 5.0
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/patches/master/freeimage/gcc5.diff"
```

Comment:
You don't need to use the full SHA1, git will accept 7/8 digits of that just fine. Otherwise :+1:.